src/Microsoft.DotNet.Tools.Test/TestRunners/AssemblyTestRunnerResolver.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.Tools.Test
+{
+    public class AssemblyTestRunnerResolver : ITestRunnerResolver
```

Comment:
nit: I had slight confusion that this class was a `RunnerResolver` but yet only returned the name of the runner. What do you think of a slight name change to `AssemblyTestRunnerNameResolver` ?